### ISSUE TITLE
WindRose chart legend

### ISSUE THREAD

**Issue by areax99** [poblabs/weewx-belchertown#401]
I upgraded from version 1.1 to version 1.2. I had set the compass rose as described in chart wiki page:
[[roseplt]]
    title = Rosa dei Venti
    [[[windRose]]]
        beauford0 = "#1278c8"
        beauford1 = "#1fafdd"
        beauford2 = "#71bc3c"
        beauford3 = "#ffae00"
        beauford4 = "#ff7f00"
        beauford5 = "#ff4500"
        beauford6 = "#9f00c5"

but I get this:

![Schermata 2020-09-12 alle 19 35 04](https://user-images.githubusercontent.com/70453504/93001595-5914eb80-f530-11ea-99d3-e9fdbf51efde.png)

In version 1.1 it worked

**Comment by poblabs** [comment]
The chart wiki says the beauford0-6 colors are optional. Remove them and it should be back to normal

**Comment by areax99** [comment]
I don't understand your answer. I don't want to remove the legend, but leave it. The problem is that all the elements of the legend are black.

**Comment by poblabs** [comment]
Remove those beauford color groups and it will be fixed how you want.

**Comment by areax99** [comment]
Sorry, but I had mistakenly deleted the quotes at the end of the color code. And the color of the legend was black
